snapcraft.yaml

```diff
@@ -59,9 +59,9 @@ parts:
   curtin:
     plugin: nil
 
-    source: https://git.launchpad.net/curtin
+    source: https://git.launchpad.net/~ogayot/curtin
```

Comment:
leaving a note here so that we use the right remove and revision before merging.